src/components/views/SettingsView.js

```diff
@@ -101,6 +110,11 @@ class SettingsView extends React.PureComponent {
         </div>
         <div style={styles.label}>Username:</div>
         <input style={styles.input} value={this.state.username} onChange={this.onUsernameChange} />
+        <div style={styles.label}>In-Game Chat View:</div>
+        <select style={styles.input} value={this.state.emuchat} onChange={this.onEmuChatChange}>
```

Comment:
Please use a checkbox instead of a select:
https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/checkbox

Reply:
I don't know how. Any attempt I have made in creating the checkbox results in it not working.